autofs-5.1.0 - fix signed comparison in inet_fill_net()

From: Ian Kent <raven@themaw.net>

Fix the less than 0 comparison of an incorrectly declared var in
inet_fill_net().
---
 CHANGELOG        |    1 +
 lib/parse_subs.c |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -225,6 +225,7 @@
 - fix config entry read buffer not checked.
 - fix FILE pointer check in defaults_read_config().
 - fix memory leak in conf_amd_get_log_options().
+- fix signed comparison in inet_fill_net().
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/lib/parse_subs.c
+++ autofs-5.0.5/lib/parse_subs.c
@@ -378,7 +378,7 @@ unsigned int get_proximity(struct sockad
 static char *inet_fill_net(const char *net_num, char *net)
 {
 	char *np;
-	unsigned int dots = 3;
+	int dots = 3;
 
 	if (strlen(net_num) > INET_ADDRSTRLEN)
 		return NULL;
